simulation/traffic_simulator/src/math/hermite_curve.cpp

```diff
@@ -338,10 +341,10 @@ double HermiteCurve::get2DCurvature(double s, bool autoscale) const
   return (x_dot * y_dot_dot - x_dot_dot * y_dot) / std::pow(x_dot * x_dot + y_dot * y_dot, 1.5);
 }
 
-double HermiteCurve::getMaximu2DCurvature() const
+double HermiteCurve::getMaximum2DCurvature() const
 {
   std::vector<double> curvatures;
-  for (double s = 0; s <= 1; s = s + 0.01) {
+  for (double s = 0; s <= 1; s = s + 0.1) {
```

Comment:
Why this magic number was modified?

Reply:
It is not a good implementation, but calculating maximum curvature by using optimization is high cost, so I decide to use magic number first.
And, this magic number describes the resolution of the curvature sampling point and in order to reduce calculation cost, I decide to modify this number.

Reply:
Leave the meaning of the number 0.1 as a comment in your code.

Reply:
Thanks, I fixed.